osquery/CMakeLists.txt

```diff
@@ -106,10 +106,10 @@ if(WINDOWS)
   ADD_OSQUERY_LINK_CORE("Wtsapi32.lib")
   ADD_OSQUERY_LINK_CORE("wbemuuid.lib")
   ADD_OSQUERY_LINK_CORE("taskschd.lib")
-  ADD_OSQUERY_LINK_CORE("libboost_system-vc140-mt-${WB_KEY}-1_63")
-  ADD_OSQUERY_LINK_CORE("libboost_regex-vc140-mt-${WB_KEY}-1_63")
-  ADD_OSQUERY_LINK_CORE("libboost_filesystem-vc140-mt-${WB_KEY}-1_63")
-  ADD_OSQUERY_LINK_CORE("libboost_context-vc140-mt-${WB_KEY}-1_63")
+  ADD_OSQUERY_LINK_CORE("libboost_system-mt-${WB_KEY}")
```

Comment:
I think that's part of this actually, I'm dropping the `vc140` and `1_63` with the new way I'm building boost libs.